<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="table">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>4</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Столбцы для рассчетов</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="col_one">
            <item>
             <property name="text">
              <string>1 столбец</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2 столбец</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3 столбец</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4 столбец</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="col_two">
            <item>
             <property name="text">
              <string>1 столбец</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2 столбец</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3 столбец</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4 столбец</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="update">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Обновить рассчеты</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>180</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="expected_label">
      <property name="text">
       <string>Математическое ожидание:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="interval_expected_label">
      <property name="text">
       <string>Доверительный интервал для математического ожидания:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dispersion_label">
      <property name="text">
       <string>Дисперсия:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="interval_dispersion_label">
      <property name="text">
       <string>Доверительный интервал для дисперсии:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="t_criterion">
      <property name="text">
       <string>T-критерий:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="hypothesis_know_d">
      <property name="text">
       <string>Гипотеза о равенстве математических ожиданий(дисперсии известны):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="z_criterion">
      <property name="text">
       <string>Z-критерий:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="hypothesis_not_know_d">
      <property name="text">
       <string>Гипотеза о равенстве математических ожиданий(дисперсии неизвестны):</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
